packages/financial-templates-lib/src/price-feed/CryptoWatchPriceFeed.js

```diff
@@ -252,11 +275,21 @@ class CryptoWatchPriceFeed extends PriceFeedInterface {
         // Sorts the data such that the oldest elements come first.
         return a.openTime - b.openTime;
       });
+  }
 
-    // 5. Store results.
-    this.currentPrice = newPrice;
-    this.historicalPricePeriods = newHistoricalPricePeriods;
-    this.lastUpdateTime = currentTime;
+  _computeTwap(endTime, ohlcs) {
+    // Combine open and close to get more data fidelity at the edges of the range.
+    const priceTimes = lodash.flatten(
```

Comment:
`flat()` is built into arrays fyi, also awesome way of creating the price array

Reply:
oh good to know -- will use that instead!

Reply:
Done!